md2conf/converter.py

```diff
@@ -115,20 +125,64 @@ def _change_ext(path: str, target_ext: str) -> str:
 class ConfluenceStorageFormatConverter(NodeVisitor):
     "Transforms a plain HTML tree into the Confluence storage format."
 
+    path: str
     base_path: str
     links: List[str]
     images: List[str]
+    page_metadata: Dict[str, ConfluencePageMetadata]
 
-    def __init__(self, base_path: str) -> None:
+    def __init__(self, path: str, page_metadata: Dict[str, ConfluencePageMetadata] = dict()) -> None:
         super().__init__()
-        self.base_path = base_path
+        self.path = path
+        self.base_path = os.path.dirname(path)
         self.links = []
         self.images = []
+        self.page_metadata = page_metadata
 
     def _transform_link(self, anchor: ET.Element) -> ET.Element:
         url = anchor.attrib["href"]
         if not is_absolute_url(url):
+            LOGGER.debug(f"not is_absolute_url = {url} from = {self.path}")
```

Comment:
I am not 100% sure I would understand what this log message means if I encountered it in a log file. Would it be better to say:
```
LOGGER.debug(f"found link {url} relative to {self.path}")
```